.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+wrapperVersion=3.3.2
+distributionType=only-script
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.8/apache-maven-3.9.8-bin.zip
```

Comment:
This version is too new and may cause problems.

Reply:
The CI runs 3.8.8. Did you test this locally @nastra?

Reply:
yes I tested this locally. If there are concerns with that Maven version then I can switch that to a version that is used by CI. Or I could update CI scripts to use the wrapper to make sure that particular maven version runs without issues

Reply:
I've updated CI to run through the wrapper to see if there would be any issues with that version

Reply:
Nice, that would be my preference. Let's see what others think.

Reply:
@joyCurry30 @Fokko CI seems to be running fine with that Maven version